subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan.c

```diff
@@ -1135,6 +1138,12 @@ static inline int isr_rx_pdu(struct lll_scan *lll, struct pdu_adv *pdu_adv_rx,
 						       FILTER_IDX_NONE,
 					 dir_report);
 		if (err) {
+			/* Auxiliary PDU LLL scanning has been setup */
+			if (IS_ENABLED(CONFIG_BT_CTLR_ADV_EXT) &&
+			    (err == -EBUSY)) {
```

Comment:
Special handling of error seems to be not needed. There is no difference between handling of failure and success of `isr_rx_scan_report`. Also `-EBUSY` is not needed to be returned. See my comment for line 1303.

Reply:
see my comment for mentioned line